src/lucky/routeable.cr

```diff
@@ -100,15 +103,18 @@ module Lucky::Routeable
 
       is_root_path = path == ""
       path = "/" if is_root_path
+      if anchor = args[:anchor]?
+        path = "#{path}##{anchor.sub(/^#/,"")}"
```

Comment:
I don't think we should sub out `#` because it could be confusing to users. I think they should sub it out themselves if they need. For example:

```crystal
link "Something", to: Posts::Show.with(the_post_id, anchor: "#foo" # Would strip out the #

# In the Posts::ShowPage
div id: "#foo" # The # is not stripped out
```

So I think we should leave the string as-is so there isn't any confusion about how things work.

Reply:
I definitely think they are different but I’m cool to remove it and make the use more clear.